dev-tools/src/main/database/all_delete.sql

```diff
@@ -18,15 +18,15 @@ DELETE FROM atht_access_token;
 
 DELETE FROM athz_role WHERE NOT (scope_id = 1 AND id = 1);
 
-DELETE FROM athz_role_permission WHERE NOT (scope_id = 1 AND id = 1);
+DELETE FROM athz_role_permission WHERE NOT (scope_id = 1 AND role_id = 1 AND id IN (1,2,3,4,5,6,7,8,9,10,11,12,13,14,15));
```

Comment:
Hi @muros-ct ,

thank you for the clarification. Yes I think src/test/... is a more appropriate folder. 

We can create anoter pull request to do that so we don't need to change this one.